docs/developer-docs/latest/development/plugins/users-permissions.md

```diff
@@ -84,7 +84,7 @@ We are using [jsonwebtoken](https://www.npmjs.com/package/jsonwebtoken) to gener
 Available options:
 
 - `expiresIn`: expressed in seconds or a string describing a time span zeit/ms.<br>
-  Eg: 60, "2 days", "10h", "7d". A numeric value is interpreted as a seconds count. If you use a string be sure you provide the time units (days, hours, etc), otherwise milliseconds unit is used by default ("120" is equal to "120ms").
+  Eg: 60, "45m", "10h", "2 days", "7d", "2y". A numeric value is interpreted as a seconds count. If you use a string be sure you provide the time units (minutes, hours, days, years, etc), otherwise milliseconds unit is used by default ("120" is equal to "120ms").
```

Comment:
We need to add a notice/warning block that is **absolutely not recommended** to exceed the default of 30 days due to massive security concerns.

Reply:
(honestly even 30 days is still pushing it, but that's our default)

Reply:
Thank you for your review @derrickmehaffy. I appreciate it a lot.

I added a warning for that :

![image](https://user-images.githubusercontent.com/43397636/122183583-90486c80-ceb5-11eb-812d-b768f3807be8.png)